tests/e2e/web/specs/postSetup.spec.ts

```diff
@@ -0,0 +1,8 @@
+import { test, expect } from '@playwright/test';
+
+//the first test is to confirm setup.ts works
+
+test('should be logged and see organization link', async ({ page }) => {
+    await page.goto('/Organization');
+    await expect(page.getByRole('link', {name: 'Organization'} )).toBeVisible(); 
```

Comment:
/organization is available to non-logged-in users so we should load another page if we want to check user auth right? I would suggest the settings page for now (if not logged in, that page would redirect to the home page).

```suggestion
test('should be logged and see settings page', async ({ page }) => {
    await page.goto('/settings');
    await expect(page.getByRole('heading', {name: 'Theme'} )).toBeVisible(); 
```